app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -138,18 +151,31 @@ class Contact extends React.Component {
           secondary_telephone: yup.string().phone().max(200, 'Max length exceeded, please limit to 200 characters.'),
           primary_telephone_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
           secondary_telephone_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
-          international_telephone: yup.string().max(50, 'Max length exceeded, please limit to 50 characters.').nullable(),
+          international_telephone: yup.string().max(50, 'Max length exceeded, please limit to 50 characters.'),
           email: yup.string().email('Please enter a valid Email.').max(200, 'Max length exceeded, please limit to 200 characters.'),
           confirm_email: yup.string().oneOf([yup.ref('email'), null], 'Confirm Email must match.'),
           preferred_contact_method: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
+          alternate_contact_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
```

Comment:
I'm finding that the way the validation works on multiple tabs can be a bit confusing. Mainly that there is no visual indication of any validation issues on the Primary tab if I'm on the Alternate tab (and vice versa). For example, I could go to the Primary tab, forget to put an email, go to the Alternate tab, fill out some valid values, and then try to click Next, and the Next button will just not work, with no real indication as to why. I think we could either:
1. Not allow you to leave a tab if it has invalid values.
2. Route you to the invalid tab if you click next on a valid tab.
3. Indicate that the other tab has invalid values in some way from the valid tab.